docs/editor_manual/new_pages/creating_body_content.rst

```diff
@@ -1,13 +1,39 @@
 Creating page body content
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The majority of the body content within a page will be created within text fields.
+Wagtail supports a number of basic fields for creating content, as well as our unique Streamfield feature which allows you to construct complex layouts by combining these basic fields in any order.
 
-There are two types of text entry fields you will see when creating a page. Some, like *Page title* are basic text fields come with no formatting options. This is because all titles display the same across a single page type. Others, *Body* for example, require more formatting, and so a set of tools are provided for this purpose. These are referred to as rich text fields. 
+===========
+Streamfield
```

Comment:
I believe it should be "StreamField" everywhere, as per http://docs.wagtail.io/en/latest/topics/streamfield.html